pkg/cloudprovider/vsphere/proto/cloudprovidervsphere.proto

```diff
@@ -0,0 +1,47 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+syntax = "proto3";
+
+option java_multiple_files = true;
+option java_package = "com.vmware.cloudprovider.vsphere";
```

Comment:
just for my understanding, why we need to define the a java_package name? thanks, maybe you can add an comment here.

Reply:
Because gRPC supports multiple languages. We just happen to be using Go. Someone out there might be using Java. It's my understanding this is required to generate a Java client.